src/Buttons/__tests__/Button.test.tsx

```diff
@@ -63,4 +66,45 @@ describe('Button', function () {
       assert.strictEqual(clickSpy.callCount, 0);
     });
   });
+  context('when forwardRef prop is present', function () {
+    beforeEach(function () {
+      const ButtonRefExample = () => {
+        const ref = useRef<HTMLInputElement>(null);
+        const onButtonClick = () => {
+          setTimeout(() => {
```

Comment:
is `setTimeout` still needed in the test ?

Reply:
Ah thanks for catching that! You're right - removed through 644e818